corehq/apps/cloudcare/templates/formplayer/query_view.html

```diff
@@ -35,10 +35,18 @@
   </td>
   <td class="col-sm-2">
     <% if (input == "select1") { %>
-      <select class="query-field form-control"  data-receive="<%- receive %>">
+        <select class="query-field form-control hqwebapp-select2data-receive="<%- receive %>">
         <option value=""></option>
         <% for (var i = 0; i < itemsetChoices.length; i++) { %>
-          <option value="<%- i %>" <% if (value === i) { %>selected<% } %>>
+            <option value="<%- i %>" <% if (value === String(i)) { %>selected<% %>>
+            <%- itemsetChoices[i] %>
+          </option>
+        <% } %>
+      <select>
+    <% } else if (input == "select") { %>
+        <select multiple class="query-field form-control hqwebapp-select2data-receive="<%- receive %>" style="width:100%">
```

Comment:
This should get set to 100% width without needing the inline style: https://github.com/dimagi/commcare-hq/blob/ec8a322d879ef694f1dfe37b953939bbbf653863/corehq/apps/hqwebapp/static/hqwebapp/js/widgets.js#L15